tests/neg/i2463.scala

```diff
@@ -0,0 +1 @@
+implicit class Foo //error
```

Comment:
You need to add a space between `//` and `error`. The framework looks for the substring `// error` in the source.